pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/utilities/Utilities.java

```diff
@@ -234,4 +235,15 @@ public static boolean useStats(ReadAccessor accessor, InputData inputData) {
             return false;
         }
     }
+
+    public static boolean useVectorization(InputData inputData) {
+        boolean isVectorizedResolver = false;
+        try {
+            isVectorizedResolver = ArrayUtils.contains(Class.forName(inputData.getResolver()).getInterfaces(), ReadVectorizedResolver.class);
+        } catch (ClassNotFoundException e) {
+            LOG.error("Unable to load resolver class: " + e.getMessage());
+            return false;
```

Comment:
no need for this line, it will return at the end of the function with default false value.